go/libkb/keyfamily.go

```diff
@@ -728,6 +739,36 @@ func (ckf ComputedKeyFamily) GetActivePGPKeys(sibkey bool) (ret []*PGPKeyBundle)
 	return
 }
 
+type RevokedKey struct {
+	Key       GenericKey
+	RevokedAt *KeybaseTime
+}
+
+func (ckf ComputedKeyFamily) GetRevokedKeys() []RevokedKey {
+	var revokedKeys []RevokedKey
+	for kid := range ckf.kf.AllKIDs {
+		ki := ckf.cki.Infos[kid]
+		if ki == nil {
+			continue
```

Comment:
Oops. I think I was wrong here, causing https://github.com/keybase/client/issues/2240. It turns out this error *is* expected, because if you've ever done a sigchain reset, then your old keys are in `AllKIDs` but not in `Infos`. I'll change this to Debug.